projects/1.16/assets/structurize/structurize/lang/zh_cn.json

```
@@ -1,25 +1,1153 @@
-    "block.structurize.down_gated_cactus_oak_timber_frame": "倒门形框镶边（仙人掌木）橡木木制框架",
-    "structurize.command.ls.message.norecipient": "您尚未加入任何会话，或是其他玩家的所有消息频道均被静音。",
-    "block.structurize.green_clay_jungle_shingle": "绿色丛林木砖瓦顶",
-    "block.structurize.slate_jungle_shingle": "板岩丛林木砖瓦顶",
-    "block.structurize.side_framed_birch_dark_oak_timber_frame": "侧面镶边（白桦木）深色橡木框架",
-    "block.structurize.side_framed_horizontal_dark_oak_cactus_timber_frame": "侧面水平镶边（深色橡木）仙人掌木制框架",
-    "block.structurize.plain_oak_oak_timber_frame": "普通橡木木制框架",
-    "block.structurize.one_crossed_rl_acacia_spruce_timber_frame": "右上斜线（金合欢木）云杉木框架",
-    "block.structurize.magenta_clay_cactus_shingle": "品红色仙人掌砖瓦顶",
-    "block.structurize.plain_oak_cobble_stone_timber_frame": "普通（橡木）圆石木制框架",
-    "block.structurize.framed_acacia_dark_oak_timber_frame": "金合欢木镶边深色橡木框架",
-    "block.structurize.down_gated_oak_jungle_timber_frame": "倒门形框镶边（橡木）丛林木制框架",
-    "structurize.config.allowplayerschematics.comment": "是否允许玩家使用自己的结构图",
-    "structurize.command.ls.remove.done": "将玩家“%s” 移出了 %s 的会话。",
-    "block.structurize.side_framed_horizontal_acacia_jungle_timber_frame": "侧面水平镶边（金合欢木）丛林木制框架",
-    "block.structurize.lime_clay_cactus_shingle": "黄绿色仙人掌砖瓦顶",
-    "block.structurize.gated_oak_cobble_stone_timber_frame": "门形框镶边（橡木）圆石木制框架",
-    "structurize.command.ls.create.already": "你已经建立了一个会话。"
+  "block.structurize.side_framed_horizontal_cactus_jungle_timber_frame": "丛林木框架（仙人掌竖边&横条）",
+  "block.structurize.horizontal_plain_dark_oak_oak_timber_frame": "",
+  "block.structurize.gated_birch_jungle_timber_frame": "门形框镶边（白桦木）丛林木制框架",
+  "item.sceptersteel.badanchorpos": "错误：结构扫描工具的基准点在扫描区域之外！请 Shift 右击来移动它！",
+  "block.structurize.plain_acacia_cobble_stone_timber_frame": "普通（金合欢木）圆石木制框架",
+  "structurize.config.allowplayerschematics.comment": "是否允许玩家使用自己的结构图",
+  "block.structurize.down_gated_oak_jungle_timber_frame": "倒门形框镶边（橡木）丛林木制框架",
+  "block.structurize.framed_acacia_dark_oak_timber_frame": "金合欢木镶边深色橡木框架",
+  "block.structurize.plain_oak_cobble_stone_timber_frame": "普通（橡木）圆石木制框架",
+  "block.structurize.magenta_clay_cactus_shingle": "品红色仙人掌砖瓦顶",
+  "block.structurize.one_crossed_rl_acacia_spruce_timber_frame": "右上斜线（金合欢木）云杉木框架",
+  "block.structurize.plain_oak_oak_timber_frame": "普通橡木木制框架",
+  "block.structurize.side_framed_horizontal_dark_oak_cactus_timber_frame": "侧面水平镶边（深色橡木）仙人掌木制框架",
+  "block.structurize.side_framed_birch_dark_oak_timber_frame": "侧面镶边（白桦木）深色橡木框架",
+  "block.structurize.slate_jungle_shingle": "板岩丛林木砖瓦顶",
+  "block.structurize.green_clay_jungle_shingle": "绿色丛林木砖瓦顶",
+  "structurize.command.ls.message.norecipient": "您尚未加入任何会话，或是其他玩家的所有消息频道均被静音。",
+  "block.structurize.down_gated_cactus_oak_timber_frame": "倒门形框镶边（仙人掌木）橡木木制框架",
+  "block.structurize.framed_oak_birch_timber_frame": "橡木镶边白桦木框架",
+  "block.structurize.side_framed_horizontal_acacia_jungle_timber_frame": "侧面水平镶边（金合欢木）丛林木制框架",
+  "structurize.command.ls.remove.done": "将玩家 “%s” 移出了 %s 的会话。",
```

|Feedback Priority|
|---|
|🌫 Dust / Non-blocking, “take it or leave it”|

```suggestion
  "structurize.command.ls.remove.done": "将玩家 %s 移出了 %s 的会话。",
```